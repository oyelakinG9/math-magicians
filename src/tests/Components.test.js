import React from 'react';
import {render, screen, cleanup, fireEvent,} from '@testing-library/react';
import Calculator from '../components/calculator';

it('5 + 4', () => {
  render(<Calculator />);

  fireEvent.click(screen.getByText('5'));
  fireEvent.click(screen.getByText('+'));
  fireEvent.click(screen.getByText('4'));
  fireEvent.click(screen.getByText('='));

  expect(screen.getByText('9')).toMatchSnapshot();
});

afterEach(cleanup);