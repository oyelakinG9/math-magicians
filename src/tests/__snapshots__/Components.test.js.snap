// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`5 + 4 1`] = `
<button
  name="9"
  type="button"
>
  9
</button>
`;
